app/src/main/java/cm/aptoide/pt/promotions/ClaimPromotionDialogFragment.java

```diff
@@ -327,6 +330,18 @@ public void onResume() {
     this.dismiss();
   }
 
+  @Override public Observable<ClaimDialogResultWrapper> dismissGenericMessage() {
```

Comment:
This method name is strange, shouldn't be tied to the dismiss logic since this is just a click listener with a mapper.
Now that I look into it the old method was probably what led to this - should we change both?